config/base/params.env

```diff
@@ -1,3 +1,3 @@
-trustyaiServiceImage=quay.io/trustyai/trustyai-service:latest
-trustyaiOperatorImage=quay.io/trustyai/trustyai-service-operator:latest
-oauthProxyImage=registry.redhat.io/openshift4/ose-oauth-proxy:latest
\ No newline at end of file
+trustyaiServiceImage=quay.io/trustyai/trustyai-service:v0.10.1
+trustyaiOperatorImage=quay.io/trustyai/trustyai-service-operator:v1.16.0
```

Comment:
not seeing this v1.16.0 in https://quay.io/repository/trustyai/trustyai-service-operator?tab=tags

Reply:
@zdtsw The pipeline will build the v1.16.0 when I apply the release tag (after merging this).

Reply:
Tag applied and image in https://quay.io/repository/trustyai/trustyai-service-operator?tab=tags